src/Serializer/AbstractItemNormalizer.php

```diff
@@ -419,7 +419,7 @@ protected function setAttributeValue(object $object, string $attribute, mixed $v
      *
      * @throws InvalidArgumentException
      */
-    protected function validateType(string $attribute, Type $type, mixed $value, string $format = null): void
+    protected function validateType(string $attribute, Type $type, mixed $value, string $format = null, array &$context): void
```

Comment:
**issue:** It's a BC break (add a default value?)

Reply:
it doesn't have to be a reference also